components/support/nimbus-fml/src/backends/kotlin/templates/FeatureManifestTemplate.kt

```diff
@@ -149,4 +165,6 @@ object {{ nimbus_object }} : FeatureManifestInterface<{{ nimbus_object }}.Featur
 {{- code }}
 {%- endfor %}
 
-{% import "macros.kt" as kt %}
\ No newline at end of file
+{% import "macros.kt" as kt %}
+{% import "java.util.Collections.emptyList"%}
+{% import "java.util.ArrayList"%}
```

Comment:
[Askama's `import` directive](https://djc.github.io/askama/template_syntax.html?highlight=import#macros) is a little different to Kotlin; I don't _think_ you need to add anymore Kotlin imports, but you can do it with [`FeatureCodeDeclaration`](https://github.com/mozilla/application-services/blob/f2e7ee575bdd35be3c61111a767aa83484c25b8c/components/support/nimbus-fml/src/backends/kotlin/gen_structs/feature.rs#L40).